beacon_node/src/config.rs

```diff
@@ -127,10 +127,6 @@ pub fn get_config<E: EthSpec>(
             client_config.http_api.allow_origin = Some(allow_origin.to_string());
         }
 
-        if let Some(fork_name) = clap_utils::parse_optional(cli_args, "http-spec-fork")? {
-            client_config.http_api.spec_fork_name = Some(fork_name);
```

Comment:
sorry, one more change

when we deprecate a flag we usually make it so that Lighthouse logs a deprecation warning when the flag is provided

we should do that for both flags, and also keep the tests where we supply the flag, but just assert that Lighthouse doesn't crash (rather than asserting anything about the value of the config)